opacclient/libopac/src/main/java/de/geeksfactory/opacclient/objects/AccountItem.java

```diff
@@ -0,0 +1,148 @@
+package de.geeksfactory.opacclient.objects;
+
+public abstract class AccountItem {
+    private long account;
+    private String title;
+    private String author;
+    private String format;
```

Comment:
I left it as a String because I don't know if we can detect `SearchResult.MediaType`s using things like CSS classes or icon URLs in the account views of APIs currently using this property or if some simply have a text describing the media type. If the latter is the case, we would need something to convert names of media types to their `SearchResult.MediaType` values to be able to show them in the app.